packages/checkout-ui-extensions/src/components/Grid/content/guidelines.md

```diff
@@ -0,0 +1,3 @@
+## Related components
+
+- [`GridItem`](https://github.com/Shopify/ui-extensions/tree/main/packages/checkout-ui-extensions/src/components/GridItem): GridItem can be used as children of Grid. It offers a way to span the element across a number of columns and rows.
```

Comment:
You don't need to repeat the component name since this content pertains to that component and you have the lead in `:`

You've got an ambiguous pronoun reference here. What does `It` stand for in the last sentence?